fenix/app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,10 @@
        Common myths about private browsing
     </string>
 
+    <!-- Total Private Browsing Mode -->
+    <string name="felt_privacy_desc_card_title" translatable="false">Leave no traces on this device.</string>
+    <string name="felt_privacy_desc_card_subtitle" translatable="false">Firefox deletes your cookies, history, and site data when you close all your private windows. Who might be able to see my activity?</string>
```

Comment:
We want to avoid the `Firefox` app name in our strings since it could be "Firefox Nightly", etc depending on the product flavor the user is using. We need to use a parameter `%1$s` that takes in the `app_name`. Also, we need to describe what the parameter represents in the localization comment.

It escapes me right now where I could find all our localization rules, but I will back track and get that to you.

Reply:
I reached out to @flodolo apparently such a localization rule is not written down anywhere. Here's a snippet of the rationale:

"The rationale is that we want folks to be able to recompile the code, and distribute it with a different name, without having to go through hoops.From a localization perspective, the only benefit is that we avoid translators changing the brand name (transliteration in other scripts). "

I will follow up with updating our docs to get this in writing somewhere.